src/charts/charts.js

```diff
@@ -188,7 +187,6 @@ define(["jquery","charts/chartingRequestMap", "websockets/binary_websockets", "w
                 type : options.type,
                 delayAmount : options.delayAmount
             });
-
```

Comment:
@4p00rv Looks like this file change is not needed. Please revert. 
Its better to submit files which really has some change so that I get an idea of how many files you touched for particular task. This gives me some idea of impact of such PR